httpclient/src/main/java/com/okta/sdk/impl/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -454,6 +465,10 @@ private void pauseBeforeRetry(int retries, HttpResponse httpResponse, long timeE
         }
     }
 
+    private RestException failedToRetry() {
+        return new RestException("Cannot retry request, next request will exceed retry configuration.");
```

Comment:
May be nice to give some more context here....

"Cannot retry request, next request will exceed the retry configuration of XX.  You have been retrying for XX.

Not a blocker though